src/components/PrivateRouter/PrivateRouter.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router';
+import { useSelector } from 'react-redux';
+
+export const PrivateRoute = ({ component: Component, ...rest }) => {
+	const admin = useSelector((state) => state.authentication.email);
+	return (
+		<Route
+			{...rest}
+			render={(props) =>
+				admin === 'admin@email.com' ? (
```

Comment:
I'd recommend to check user `role` instead of user `email`

Reply:
Additionally, I'd move this check above, i.e:
```
isLoggedIn ? (<Route ... />) : (<Redirect ... />)
```
In this case, your `PrivateRoute` component can use any `Route` prop (`component`, `render` or `children`)